pages/lostfound.tsx

```diff
@@ -0,0 +1,180 @@
+    </Alert>
+    <br />
+    <TextInputField
+      label="Name of Item Lost"
+      isInvalid={nameInvalid}
+      required
+      placeholder={"Item"}
+      validationMessage = {nameInvalid ? "Must have an item name" : null}
+      value={name}
+      onChange={e => setName(e.target.value)}
+    />
+    <TextInputField
+      label="Displayed Sender Name"
+      required
+      isInvalid={senderInvalid}
+      description={senderNameDesc}
+      placeholder={isLoading ? "Tammy Tiger": user.name}
+      validationMessage = {senderInvalid ? "Must have sender name" : null}
+      value={sender}
+      onChange={e => setSender(e.target.value)}
+    />
+    <TextInputField
+    label="Description of Item"
+    isInvalid={descInvalid}
+    required
+    placeholder={"Description"}
+    validationMessage = {descInvalid ? "Must have a description" : null}
+    value={desc}
+    onChange={e => setDesc(e.target.value)}
+    />
+    <Pane>
+      <Button size="large" appearance="primary" float="right">
+        Submit
+      </Button>
+      <Button 
+        size="large" 
+        appearance="default"
+        float="right"
+        marginRight="8px"
+      >
+        Cancel
+      </Button>
+      <Link href="/">
+        <Button size="large" float="left">Back</Button>
+      </Link>
+            
+    </Pane>
+    <br/>
+    <Dialog
+      isShown={showConfirm}
+      hasHeader={false}
+      hasClose={false}
+      onConfirm={async () => {
+        await onSend({sender, name, desc});
+        setShowConfirm(false);
+      }}
+      onCloseComplete={() => setShowConfirm(false)}
+      confirmLabel="Send Email"
+      intent="warning"
+    >
+      <Pane
+        marginTop={35}
+        marginBottom={20}
+        fontFamily="Nunito"
+        display="flex"
+        alignItems="center"
+      >
+        <InfoSignIcon marginRight={10} /> You are about to add your message to the weekly Hoagie Mail Digest service.
+      </Pane>
+      <Text>
+      Once you click "Submit", Hoagie will append your message in the upcoming weekly digest email. This is sent to <b>all residential college listservs on your behalf</b>. 
+      Your NetID will be included with your message regardless of the content.
+      </Text>
+      <Alert
+        intent="danger"
+        marginTop={20}
+        >
+          Effective immediately, <b>you SHOULD NOT use Hoagie Mail to send emails about lost and found items or items you are selling.</b><br />
+          Violation of this rule will result in a ban.
+        </Alert>
+      <Alert
+        intent="warning"
+        title="Use this tool responsibly"
+        marginTop={20}
+        >
+          If Hoagie Mail Digest is used to send offensive,
+          intentionally misleading or harmful messages, the user will be banned from the platform
+          and, if necessary, reported to the University.
+        </Alert>
+    </Dialog>
+  </Pane>
+
+  const SuccessPage = <Pane 
```

Comment:
please remove the entirety of SuccessPage into a new component called SuccessPage.tsx and import it here instead, it'd be good if you also updated the default form to use the same component.